### PR TITLE
Improved Exception Handling in Tests cases

### PR DESCRIPTION
**Pull request by Siedlerchr**
Never catch an Exception
Removed unnecessary Exceptions
Throw specific Exceptions, never only Exception
Ref #1012 

- [x] Changes in pull request outlined? (What, why, ...)
- [x] Tests created for changes?
- [x] Tests green?

### CODE REVIEW

File: src/test/java/net/sf/jabref/BibtexTestData.java [hunk 1]
```diff
@@ -32,11 +31,9 @@ public static BibDatabase getBibtexDatabase() {
 
         BibtexParser parser = new BibtexParser(reader);
         ParserResult result = null;
-        try {
+
             result = parser.parse();
```
Comment (oscargus) [inline src/test/java/net/sf/jabref/BibtexTestData.java]: Assign directly.

File: src/test/java/net/sf/jabref/FileBasedTestCase.java [hunk 2]
```diff
@@ -35,10 +36,9 @@ public void setUp() throws Exception {
         database = BibtexTestData.getBibtexDatabase();
         entry = database.getEntries().iterator().next();
 
-        // Create file structure
-        try {
+
             root = FileBasedTestHelper.createTempDir("UtilFindFileTest");
```
Comment (oscargus) [inline src/test/java/net/sf/jabref/FileBasedTestCase.java]: Indentation

**Comment by oscargus** [comment]
Good work! :-+1:

**Comment by stefan-kolb** [comment]
@Siedlerchr Btw, you should assign your commit email with your Github account, see https://github.com/JabRef/jabref/pull/1014/commits/758e7c2d6fe6cb9e8a82f4bfda2f601608647344

**Comment by Siedlerchr** [comment]
@oscargus Fixed.
@stefan-kolb I just added my git mail address to my github account as secondary mail address.

**Comment by Siedlerchr** [comment]
Was done by Eclipse. Will change this back

**Pull request merged**